arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-preparing-node-config.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'How to Configure a Node Using the Orbit SDK'
+sidebar_label: 'Node Config Generation'
+description: 'How to configure a node using the Orbit SDK'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 4
+---
+
+This guide will walk you through configuring an Orbit node supporting a <a data-quicklook-from="arbitrum-rollup-chain">Rollup</a> or <a data-quicklook-from="arbitrum-anytrust-chain">Anytrust</a> chain.
+
+- Prerequisite: having deployed an Orbit chain. If you haven't done so yet, you can find detailed instructions in the [Rollup Deployment Parameters](orbit-sdk-deploying-rollup-chain#rollup-deployment-parameter) section of the rollup deployment guide.
```

Comment:
```suggestion
- Prerequisite: having deployed an Orbit chain. If you haven't done so yet, you can find detailed instructions in the [Orbit rollup deployment](orbit-sdk-deploying-rollup-chain) guide.
```
I think?

Reply:
Yes, it's much clearer that way.
Merged.